USERS.md

```diff
@@ -253,3 +253,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Yubo](https://www.yubo.live/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Trusting Social](https://trustingsocial.com/)
```

Comment:
@cuong-ts can you move this up for alphabetical order?

Reply:
fixed @crenshaw-dev